<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entering</class>
 <widget class="QDialog" name="Entering">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QPushButton" name="k5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>322</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>198</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>198</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="k8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>446</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k_del">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>570</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>446</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>1281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>322</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k_dot">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>198</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>322</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>446</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k0">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>570</y>
     <width>161</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px rgb(173, 173, 173);
border-radius: 10px;
padding: 0 8px;
background: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>110</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>60</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>711</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>60</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>INSERT PASSWORD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
